### PR TITLE
Always copy test data out from jar file

### PR DESCRIPTION
**Pull request by fx19880617**
a quick fix for issue of referring to jar uri.
```
Exception in thread "main" java.lang.IllegalStateException: PinotFS for scheme: jar has not been initialized

at shaded.com.google.common.base.Preconditions.checkState(Preconditions.java:518)

at org.apache.pinot.spi.filesystem.PinotFSFactory.create(PinotFSFactory.java:78)

at

org.apache.pinot.plugin.ingestion.batch.standalone.SegmentGenerationJobRunner.run(SegmentGenerationJobRunner.java:115)

at org.apache.pinot.spi.ingestion.batch.IngestionJobLauncher.kickoffIngestionJob(IngestionJobLauncher.java:96)

at org.apache.pinot.spi.ingestion.batch.IngestionJobLauncher.runIngestionJob(IngestionJobLauncher.java:77)

at org.apache.pinot.tools.admin.command.QuickstartRunner.launchDataIngestionJob(QuickstartRunner.java:183)

at org.apache.pinot.tools.Quickstart.execute(Quickstart.java:154)

at org.apache.pinot.tools.Quickstart.main(Quickstart.java:209)
```

### CODE REVIEW

**Comment by snleee** [comment]
Verified that `quick-start-batch.sh` is working fine on my mac laptop.

File: pinot-tools/src/main/java/org/apache/pinot/tools/utils/JarUtils.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+public class JarUtils {
+  private JarUtils() {
+  } // non-instantiable
+
+  /**
+   * Copies a directory from a jar file to an external directory.
+   */
+  public static void copyResourcesToDirectory(String fromJarFilePath, String jarDir, String destDir)
+      throws IOException {
+    if (fromJarFilePath.startsWith("jar:")) {
```
Comment (snleee) [inline pinot-tools/src/main/java/org/apache/pinot/tools/utils/JarUtils.java]: lets put `private static final String JAR_PREFIX = "jar:"`

File: pinot-tools/src/main/java/org/apache/pinot/tools/utils/JarUtils.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      fromJarFilePath = fromJarFilePath.substring(4);
```
Comment (snleee) [inline pinot-tools/src/main/java/org/apache/pinot/tools/utils/JarUtils.java]: substring(JAR_PREFIX.length())

File: pinot-tools/src/main/java/org/apache/pinot/tools/utils/JarUtils.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    }
+    if (fromJarFilePath.startsWith("file:")) {
```
Comment (snleee) [inline pinot-tools/src/main/java/org/apache/pinot/tools/utils/JarUtils.java]: same here

File: pinot-tools/src/main/java/org/apache/pinot/tools/utils/JarUtils.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+          }
+          try {
+            out.close();
+          } catch (IOException ignored) {
+          }
+        }
+      }
+    }
+  }
+}
```
Comment (snleee) [inline pinot-tools/src/main/java/org/apache/pinot/tools/utils/JarUtils.java]: add line

**Comment by siddharthteotia** [comment]
Verified this works on Mac. The error reported earlier is not seen anymore.

Thanks @fx19880617  for fixing this.

**Pull request merged**